test/AutoDelegateBravoGovernor.t.sol

```diff
@@ -4,14 +4,28 @@ pragma solidity ^0.8.23;
 import {Test} from "forge-std/Test.sol";
 import {GovernorBravoDelegateMock} from "./mocks/GovernorBravoDelegateMock.sol";
 import {IGovernorBravoDelegate} from "../src/interfaces/IGovernorBravoDelegate.sol";
-import {FakeAutoDelegateBravoGovernor} from "./fakes/FakeAutoDelegateBravoGovernor.sol";
+import {AutoDelegateBravoGovernor} from "../src/auto-delegates/extensions/AutoDelegateBravoGovernor.sol";
+
+contract InstanceOfAutoDelegateBravoGovernor is AutoDelegateBravoGovernor {
```

Comment:
Generally we refer to these kinds of contracts as "harnesses". This should go in the `harnesses/` subdir and be named `AutoDelegateBravoGovernorHarness`

Reply:
Done: [4b1c791](https://github.com/withtally/stGOV/pull/87/commits/4b1c79156ab86ab471bee1bbd11bd4e268695d39)